### ISSUE TITLE
Add analytics to Storybook

### ISSUE THREAD

**Issue by cvalarida** [department-of-veterans-affairs/vets-design-system-documentation#1139]
## Issue Description
To capture usage data on Storybook, we need to add the [Google Analytics Addon](https://storybook.js.org/addons/@storybook/addon-google-analytics).

## Questions
Is the GA ID sensitive information? Do we need to add that into the Storybook deploy via CI?

**Comment by caw310** [comment]
@jamigibbs @Andrew565 - is this still relevant?

**Comment by jamigibbs** [comment]
@caw310 Yes. There is no evidence that we've added analytics to Storybook yet so this should probably stay open if we still want to do this.
